docs/source/algorithms/Comment-v1.rst

```diff
@@ -0,0 +1,41 @@
+
+.. algorithm::
+
+.. summary::
+
+.. alias::
+
+.. properties::
+
+Description
+-----------
+
+This simple algorithm just adds a comment record to the history of a workspace and is a simple way of annotating your workflow.
+
```

Comment:
I'd suggest adding a line here saying that when you export a python script or IPython notebook, these comment records will be included as a `#` comment at the point where the algorithm was run. This is not/cannot be shown in the example and might not be obvious to users.